All-Schuck-Up/AD-410-F
# Ryan

This is quite a big update, but with that in mind I added a text file "changelog.txt" into the branch for a quick summary of what I added and the order.

1. To run the app $ npm run server
    -should prompt in terminal with 'server started on port 5000', 'mongoDB connected...'

2. To test create user route, I use postman 
    - Its a "POST" request to http://localhost:5000/api/users
    - set "HEADERS" key: content-type, value: application/json
    - in the "BODY" tab, select raw and pass in the following fields { "name":  "_______", 
    "email":  "_______", "password": "123456" }
    - for a response, if successful should provide a token, if error will display error

3. To test login user, again use postman
   - its a "POST" request to http://localhost:5000/api/auth
    - set "HEADERS" key: content-type, value: application/json
    - in the "BODY" tab, select raw and pass in the following fields { "email":  "_______", "password": 
    "_______" } **must match a user in database***
    - if successful log in postman will provide token (copy token and go to "jwt.io" to verify login info)
